spring-xd-integration-test/src/test/java/org/springframework/xd/integration/test/FileJdbcTest.java

```diff
@@ -37,6 +39,8 @@
 public class FileJdbcTest extends AbstractJobTest {
 
 	private final static String DEFAULT_FILE_NAME = "filejdbctest";
+	private final static String DEFAULT_DIRECTORY = XdEnvironment.RESULT_LOCATION +
```

Comment:
This looks like it was pulled in with some other unrelated changes under integration-test project